roles/openshift_node/tasks/install.yml

```diff
@@ -107,26 +107,8 @@
         Please ensure repos are configured properly to provide these packages
         and indicated versions.
 
-- name: Enable the CRI-O service
-  systemd:
-    name: "crio"
-    enabled: yes
```

Comment:
did we intend to delete the crio service here?

Reply:
definitely!
fixing